circle.yml

```diff
@@ -114,5 +157,65 @@ workflows:
   version: 2
   build-test-and-deploy:
     jobs:
-      - build
+      - build:
+          name: quality_migrations_tests
+          tests_group: 0
+          tests_type: clean test.quality test.migrations_missing
+      - build:
+          name: python_unit_tests
+          tests_group: 0
+          tests_type: clean test.unit
+          requires:
+            - quality_migrations_tests
+      - build:
+          name: browser_tests
+          tests_group: 0
+          tests_type: clean test.browser
+          requires:
+            - quality_migrations_tests
+      - build:
+          name: js_tests
+          tests_group: 0
+          tests_type: clean test.js
+          requires:
+            - quality_migrations_tests
+      - build:
+          name: integration_tests_group_1
+          tests_group: 1
+          tests_type: test.integration
+          requires:
+            - python_unit_tests
+            - browser_tests
+            - js_tests
+      - build:
+          name: integration_tests_group_2
+          tests_group: 2
+          tests_type: test.integration
+          requires:
+            - python_unit_tests
+            - browser_tests
+            - js_tests
+      - build:
+          name: integration_tests_group_3
+          tests_group: 3
+          tests_type: test.integration
+          requires:
+            - python_unit_tests
+            - browser_tests
+            - js_tests
+      - coverage:
+          name: coverage
+          requires:
+            - integration_tests_group_1
+            - integration_tests_group_2
+            - integration_tests_group_3
+  cleanup-workflow:
+    triggers:
+      - schedule:
+          cron: "0 1 * * *"
+          filters:
+            branches:
+              only:
+                - master
```

Comment:
@viadanna, why is the cleanup job running only on the master branch? It should run in all the CI runs to clean up resources older than 8 hours.

Reply:
It's now running on a cron, everyday at 1am, so there's no need to run on all branches given the script will cleanup all resources, @lgp171188

Reply:
@viadanna, so there is a separate CI job? Nice! :+1: